docs/volto_customization/installation.md

```diff
@@ -17,150 +17,124 @@ Getting started with Volto involves setting up a development environment, unders
+- <a target="_blank" href="https://www.gnu.org/software/make/">GNU make</a>
 - <a target="_blank" href="https://www.docker.com/get-started">Docker</a> (if using the Plone docker images - <a target="_blank" href="https://6.docs.plone.org/install/containers/index.html">see instructions for installation and usage</a>)
 
 The versions of Python that are supported in Volto depend on the version of Plone that you use.
 
-| Plone | Python       | Volto        |
-| ----- | ------------ | ------------ |
-| 6.0   | 3.8-3.11     | 16.0 or 17.0 |
-| 5.2   | 2.7, 3.6-3.8 | 15.0         |
+| Plone | Python       | Volto          |
+| ----- | ------------ | -------------- |
+| 6.1   | 3.10-3.12    | 18.0           |
+| 6.0   | 3.8-3.12     | 16.0,17.0,18.0 |
+| 5.2   | 2.7, 3.6-3.8 | 15.0           |
 
 Depending on the operating system that you are using, some of the following pre-requisites might change.
 They assume you have a macOS/Linux machine.
 
-## Bootstrap a new Volto project
+## Bootstrap a new Plone stack
 
-To bootstrap a new Volto project, you can use Yeoman [@plone/generator-volto](https://github.com/plone/generator-volto).
-First, install it as a global tool (<a target="_blank" href="https://6.docs.plone.org/volto/recipes/creating-project.html">see instructions for installation</a>):
+To bootstrap a new Plone project(with both backend and frontend), you can use [Cookieplone](https://github.com/plone/cookieplone).
+You can use pipx to run cookieplone (<a target="_blank" href="https://6.docs.plone.org/install/create-project-cookieplone.html#generate-the-project">see instructions for installation</a>):
 
 ```shell
-npm install -g yo
-npm install -g @plone/generator-volto
+pipx run cookieplone project
 ```
 
-Then you can bootstrap the project with:
+You will be presented with a series of prompts. You can also specify the addons you want to install along with the project. You can type the name of the project as `tutorial-project` and name of the addon as `volto-teaser-tutorial` when prompted for the addons.
 
 ```shell
-yo @plone/volto volto-tutorial-project
+[11/17] Volto Addon Name (volto-project-title): volto-teaser-tutorial
 ```
 
-The yo-based generator partially integrates add-ons (it can generate a
-`package.json` with add-ons and workspaces already specified). When prompted
-to add add-ons, choose `false`.
+You can accept the rest of the default values in square brackets (`[default-option]`) by hitting the {kbd}`Enter` key, or enter your preferred values.
+For the training, we will use the default values for the rest of the prompts.
 
-Now you can start your newly created Volto project:
+## Install the project
 
-```shell
-cd volto-tutorial-project
-yarn start
-```
+To work on your project, you need to install both the frontend and backend.
 
-You can then login with admin/admin at <a target="_blank" href="http://localhost:3000/login">http://localhost:3000/login</a>.
+Change your current working directory to `tutorial-project`.
```

Comment:
This is the default value. Replace all instances throughout the training for consistency.

```suggestion
Change your current working directory to `project-title`.
```